web/enoent.jsp

```diff
@@ -54,7 +54,7 @@ include file="menu.jspf"
 {
     PageConfig cfg = PageConfig.get(request);
     String configError = "";
-    if (cfg.getSourceRootPath().isEmpty()) {
+    if (cfg.getSourceRootPath() == null || cfg.getSourceRootPath().isEmpty()) {
```

Comment:
I wasn't quite sure about the `errorPage="error.jsp" isErrorPage="true"` configuration as it's not really common use case however I checked that it works. Actually it makes more sense to return 500 instead of 404 with the same error message.